hisi-idt.py

```diff
@@ -85,7 +86,7 @@ class bootdownload(object):
 
     def __init__(self,chiptype,serialport):
         try:
-            self.s = serial.Serial(port=serialport, baudrate=115200, timeout=1)
+            self.s = serial.Serial(dsrdtr=True, rtscts=True, port=serialport, baudrate=115200, timeout=1)
```

Comment:
Hmnnn... looks like I missed this one yesterday (I typed it in... maybe I deleted it by mistake). Either way...

This change doesn't seem to have anything to do with python3; it is a bug fix to make pyserial work on hikey960/970 and should be a separate patch.

Reply:
Sure, will split the patches.

Thanks for your detailed review :)